reference/OpenAPI.json

```diff
@@ -7647,14 +7647,264 @@
+              "$ref": "#/components/schemas/Category"
+            }
+          },
+          "object": {
+            "type": "string",
+            "default": "campaign",
+            "description": "The type of the object represented by JSON. This object stores information about the campaign."
+          },
+          "loyalty_tiers_expiration": {
+            "$ref": "#/components/schemas/LoyaltyTiersExpirationAll"
+          },
+          "validation_rules_assignments": {
+            "$ref": "#/components/schemas/ValidationRulesAssignmentsList"
+          },
+          "access_settings_assignments": {
+            "$ref": "#/components/schemas/AccessSettingsCampaignAssignmentsList"
+          }
+        },
+        "required": [
+          "id",
+          "name",
+          "campaign_type",
+          "type",
+          "auto_join",
+          "join_once",
+          "use_voucher_metadata_schema",
+          "created_at",
+          "creation_status",
+          "vouchers_generation_status",
+          "protected",
+          "category_id",
+          "categories",
+          "object"
+        ]
+      },
+      "LoyaltyCampaignVoucher": {
+        "type": "object",
+        "description": "Schema model for a campaign voucher.",
+        "title": "Loyalty Campaign Voucher",
+        "properties": {
+          "type": {
+            "type": "string",
+            "default": "LOYALTY_CARD",
+            "enum": [
+              "LOYALTY_CARD"
+            ],
+            "description": "Type of voucher."
+          },
+          "loyalty_card": {
+            "description": "Defines the voucher loyalty card details.",
+            "allOf": [
+              {
+                "$ref": "#/components/schemas/CampaignLoyaltyCard"
+              }
+            ]
+          },
+          "redemption": {
+            "type": "object",
+            "description": "Defines the redemption limits on vouchers.",
+            "properties": {
+              "quantity": {
+                "type": "integer",
+                "nullable": true,
+                "description": "How many times a voucher can be redeemed. A `null` value means unlimited."
+              }
+            },
+            "required": [
+              "quantity"
+            ]
+          },
+          "code_config": {
+            "allOf": [
+              {
+                "$ref": "#/components/schemas/CodeConfig"
+              }
+            ],
+            "required": [
+              "length",
+              "charset",
+              "pattern"
+            ]
+          },
+          "is_referral_code": {
+            "type": "boolean",
+            "description": "Always `false` for loyalty card voucher"
+          },
+          "start_date": {
+            "type": "string",
+            "format": "date-time",
+            "description": "Activation timestamp defines when the campaign starts to be active in ISO 8601 format. Campaign is *inactive before* this date. ",
```

Comment:
Tutaj to olewam, bo to kiedyś ogarnę sam już przekrojowo w całym `OpenAPI.json`, ale jest bardzo dużo kulawych opisów z tym fragmentem "in ISO 8601 format".

Tak, jak jest tutaj, to znaczy, że "kampania będzie aktywna w formacie ISO 8601".